field_audit/management/commands/bootstrap.py

```diff
@@ -0,0 +1,130 @@
+from contextlib import contextmanager
+
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+
+from field_audit.field_audit import get_audited_models
+from field_audit.models import AuditEvent
+
+
+class Command(BaseCommand):
+
+    help = "Create bootstrap AuditEvent records for audited model classes."
+    models = {}
+
+    @classmethod
+    def setup_models(cls):
+        def model_name(model, unique=False):
+            if unique:
+                return f"{model._meta.app_label}.{model.__name__}"
+            return model.__name__
+        for model_class in get_audited_models():
+            name = model_name(model_class)
+            collision = cls.models.setdefault(name, model_class)
+            if collision is not model_class:
+                del cls.models[name]
+                cls.models[model_name(collision, unique=True)] = collision
+                cls.models[model_name(model_class, unique=True)] = model_class
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "operation",
+            choices=self.operations,
+            help="Type of bootstrap operation to perform.",
+        )
+        parser.add_argument(
+            "models",
+            choices=sorted(self.models),
+            nargs="+",
+            help="Model class(es) to perform the bootstrap operation on.",
+        )
+        parser.add_argument(
+            "--commit",
+            action="store_true",
+            default=False,
+            help=(
+                "Create the audit event records (a dry-run is performed when "
+                "this option is omitted)."
+            ),
+        )
+
+    def handle(self, operation, models, commit, **options):
+        self.stdout.ending = None
+        self.logfile = self.stdout
+        if not commit:
+            self.log_info(
+                "DRY-RUN, no events will be written to the database. Use the "
+                "--commit option to perform persistent writes."
```

Comment:
"no events will be written to the database" is not quite true, right? Events are written, but then the transaction is rolled back. This could cause some database state to change (sequences might be incremented, etc). Would it be difficult to change the command so it actually does not write to the database, or is there some value in write + rollback?

Reply:
It is _more_ difficult. Using the transaction rollback was for pure convenience (and it is convenient!). I wasn't thinking about sequences and other DB state though, now I don't like the transaction rollback as much. I'll look into this.

Reply:
Fixed in 72b9be5.